### DISCUSSION TITLE
Error Help

### DISCUSSION THREAD

**Discussion by Meditacion** [kitadai31/revanced-patches-android6-7#9] [General]
Me aparece error, paso txt. no se programación y apenas entiendo, pero hice el procedimiento ayer y fallo, según entiendo problema de servidor, hoy probé otra vez y fallo

Initializing installer
Creating working directory
Copying original apk
Unpacking input apk
Reading dex files
Decoding AndroidManifest.xml only, because resources are not needed
Merging integrations
Deleting existing resource cache directory
Decoding resources
Applied client-spoof
Applied custom-branding-icon-afn-red
Applied custom-branding-name
Applied custom-seekbar-color
Applied custom-video-speed
Applied default-video-quality
Applied default-video-speed
Applied disable-haptic-feedback
Applied enable-external-browser
Applied enable-minimized-playback
Applied enable-new-layout
Applied enable-old-quality-layout
Applied enable-open-links-directly
Applied enable-seekbar-tapping
Applied enable-tablet-miniplayer
Applied enable-wide-searchbar
Applied force-vp9-codec
Applied header-switch
Applied hide-auto-captions
Applied hide-auto-player-popup-panels
Applied hide-autoplay-button
Applied hide-button-container
Applied hide-cast-button
Applied hide-channel-watermark
Applied hide-comment-component
Applied hide-create-button
Applied hide-crowdfunding-box
Applied hide-email-address
Applied hide-endscreen-cards
Applied hide-endscreen-overlay
Applied hide-filmstrip-overlay
Applied hide-firsttime-background-notification
Applied hide-flyout-panel
Applied hide-fullscreen-panels
Applied hide-general-ads
Applied hide-info-cards
Applied hide-live-chat-button
Applied hide-mix-playlists
Applied hide-next-prev-button
Applied hide-player-captions-button
Applied hide-player-overlay-filter
Applied hide-shorts-button
Applied hide-shorts-component
Applied hide-shorts-pivot-bar
Applied hide-snackbar
Applied hide-startup-shorts-player
Applied hide-stories
Applied hide-suggested-actions
Applied hide-time-and-seekbar
Applied hide-tooltip-content
Applied hide-video-ads
Applied layout-switch
Applied microg-support
Applied optimize-resource
Applied overlay-buttons
Applied patch-options
Applied protobuf-spoof
Applied remove-player-button-background
Applied return-youtube-dislike
Applied settings
Applied sponsorblock
Applied swipe-controls
Applied switch-create-notification
Applied theme
Applied translations
Repacking patched apk
Compiling resources
An error occurred! Aborting
Error:
brut.androlib.AndrolibException: brut.common.BrutException: could not exec (exit code = 1): [/data/app/app.revanced.manager.flutter-1/lib/arm/libaapt2.so, compile, --dir, /data/user/0/app.revanced.manager.flutter/cache/patcher/tmp-NHKUJJ/cache/res, --legacy, -o, /data/user/0/app.revanced.manager.flutter/cache/patcher/tmp-NHKUJJ/cache/build/resources.zip]
at brut.androlib.res.AndrolibResources.aapt2Package(AndrolibResources.java:365)
at brut.androlib.res.AndrolibResources.aaptPackage(AndrolibResources.java:619)
at app.revanced.patcher.Patcher.save(Patcher.kt:126)
at app.revanced.manager.flutter.MainActivity.runPatcher$lambda-20(MainActivity.kt:212)
at app.revanced.manager.flutter.MainActivity.$r8$lambda$kxvfcFiIKARIH6oxYjk9gWWxjp0(MainActivity.kt)
at app.revanced.manager.flutter.MainActivity$$ExternalSyntheticLambda0.run(D8$$SyntheticClass)
at java.lang.Thread.run(Thread.java:818)
Caused by: brut.common.BrutException: could not exec (exit code = 1): [/data/app/app.revanced.manager.flutter-1/lib/arm/libaapt2.so, compile, --dir, /data/user/0/app.revanced.manager.flutter/cache/patcher/tmp-NHKUJJ/cache/res, --legacy, -o, /data/user/0/app.revanced.manager.flutter/cache/patcher/tmp-NHKUJJ/cache/build/resources.zip]
at brut.util.OS.exec(OS.java:99)
at brut.androlib.res.AndrolibResources.aapt2Package(AndrolibResources.java:361)
... 6 more

**Comment by Meditacion** [comment]
Probaré otra vez con menos parches

**Comment by kitadai31** [comment]
Is your device Android 6.0?

**Comment by Meditacion** [comment]
Pude conseguirlo, encontre un tipo que lo parcheo y lo subio a Google
Drive, lo descargue instale y hasta ahora funciona.Creo que se basó en tu
trabajo

Paso link del tipo:

https://www.reddit.com/user/Snowblind_Wolf/
